WordPress/gutenberg
# [RNMobile] Add ruby version file.

This will 

Use same version of ruby that is used on WP iOS



## What?

Adds a `.ruby-version` file to the `ios` directory of the react native editor package.

## Why?

To allow the ruby manager `rbenv` to switch the right version when setting up ios.

## How?

Adds a `.ruby-version` file set to use the same version of ruby used by WP iOS .

## Testing Instructions




1. Install ruby `^3.0.0` and set as the default ruby e.g.`rbenv install -s 3.0.0 && rbenv global 3.0.0` ( cocoapods requires `rexml` which is no longer included in ruby `3` )
2. Try running any of the bundle dependent commands in `package/react-native-editor` e.g. `npm run preios`

if ruby `2.7.4` is not installed the command should fail with `rbenv` complaining that `2.7.4` is not installed.
if ruby `2.7.4` is installed the command should use `2.7.4` and not fail due to `3.0.0` being the global ruby version.


## Screenshots or screencast
